content/applications/finance/payment_providers/razorpay.rst

```diff
@@ -23,6 +23,11 @@ Configuration on Razorpay Dashboard
    checkboxes are ticked.
 #. Click on :guilabel:`Create Webhook` to finalize the configuration.
 
+.. note::
```

Comment:
I'd make it an `important` admonition since it's quite an important feature to miss out.
```suggestion
.. important::
```